scripts/fix-next-installation-enhanced.js

```
@@ -0,0 +1,247 @@
+  try {
+    const packageJsonPath = path.resolve(process.cwd(), 'package.json');
+    const packageJson = JSON.parse(fs.readFileSync(packageJsonPath, 'utf8'));
+    
+    // Update scripts to use npx
+    if (packageJson.scripts) {
+      if (packageJson.scripts['dev:next']) {
+        packageJson.scripts['dev:next'] = 'npx next dev';
+      }
+      
+      if (packageJson.scripts['build'] && packageJson.scripts['build'].includes('next build')) {
+        packageJson.scripts['build'] = 'npx next build --no-lint';
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)